src/ai/backend/agent/stats.py

```diff
@@ -524,12 +525,18 @@ async def collect_per_container_process_stat(
             results = await asyncio.gather(*_tasks, return_exceptions=True)
             updated_cids: Set[ContainerId] = set()
             for result in results:
-                if isinstance(result, Exception):
-                    log.error(
-                        "collect_per_container_process_stat(): gather_process_measures() error",
-                        exc_info=result,
-                    )
-                    continue
+                match result:
+                    case psutil.NoSuchProcess():
+                        log.debug(str(result))
```

Comment:
I think `psutil` is an implementation detail of the intrinsic CPUPlugin, so we should avoid importing it in the outer-level interface here.

Let's put matching of `psutil.NoSuchProcess` inside the `src/ai/backend/agent/docker/intrinsic.py`.

NOTE: We also need to backport this PR to the activator plugin that overrides the CPUPlugin in enterprise setups. I'll take care of that after this PR is done.

Reply:
👌 I will work on it